tests/test_command_parse.py

```diff
@@ -222,6 +223,10 @@ def test_crawlspider_no_matching_rule(self):
         self.assertRegex(_textmode(out), r"""# Scraped Items  -+\n\[\]""")
         self.assertIn("""Cannot find a rule that matches""", _textmode(stderr))
 
+        status, out, stderr = yield self.execute([self.url('/invalid_url')])
```

Comment:
Please, give this test its own test method instead of running it within the existing `test_crawlspider_no_matching_rule`.

Reply:
Thanks @Gallaecio , should I use also the decorator @defer.inlineCallbacks in my new test?